docs/cli/installation.md

```diff
@@ -24,7 +24,7 @@ yarn add --dev all-contributors-cli # or npm
 Init the project using `init` and answer a few questions
 
 ```console
-yarn all-contributors init # npm run
+yarn all-contributors init # or npx all-contributors init 
```

Comment:
Is there any tangible advantage in using `npx` over `npm run` in scenarios where one would use the AC CLI?

Reply:
`npm run` is not equivalent to `yarn all-contributors init`. 

`npx all-contributors init` is equivalent to it instead.

Reply:
That makes sense.